public/locales/en/translation.json

```diff
@@ -32,5 +32,67 @@
     "NAME": "NAME",
     "DATE OF BIRTH": "DATE OF BIRTH",
     "VACCINATION RECORD": "VACCINATION RECORD"
+  },
+  "footer": {
+    "FAQ": "FAQ",
+    "Privacy": "Privacy"
+  },
+  "faq": {
+    "Frequently asked questions": "Frequently asked questions",
+    "items": [
+      {
+        "question": "What is a SMART\u00AE Health Card and where can I get one?",
+        "answer": "SMART\u00AE Health Cards are paper or digital versions of your clinical information, such as vaccination history or test results. They allow you to keep a copy of your records on hand and easily share this information with others if you choose. Please visit <a href=\"https://smarthealth.cards/en/\">https://smarthealth.cards</a> for more information and details on how to obtain a SMART\u00AE Health Card."
+      },
+      {
+        "question": "What does it mean if you have an unsupported QR code?",
```

Comment:
I think it's good that we are including a question regarding the reasons for an unsupported QR code. Side note: since we are mentioning that a QR code may be unreadable or non-SHC compliant, would it beneficial to add functionality to our QR scanner to time out and direct to an error page explaining that the QR code is unreadable or non-SHC compliant?